ls1intum/Artemis
# `Quiz exercises`: Improve performance when saving submissions in synchronized quizzes

### Checklist
#### General


#### Server

### Motivation and Context
There were unnecessary many DB operations when saving quiz submissions in the synchronized mode.
The PR also addresses an exceptional case, where Artemis might try to store participations + quiz submissions multiple times into the database, but cannot recover from it and would do this on a continuous basis

### Description


### Steps for Testing
Prerequisites:
- 1 Instructor
- Multiple Students
- 1 Quiz Exercise (synchronized)

1. Create the quiz
2. Participate in it with multiple students
3. Check that everything works correctly

### Review Progress





#### Code Review
#### Manual Tests
